databind/src/main/java/tech/ydb/yoj/databind/schema/Changefeed.java

```diff
@@ -25,6 +25,21 @@
      */
     Format format() default Format.JSON;
 
+    /**
+     * Virtual timestamps
+     */
+    boolean virtualTimestampsEnabled() default false;
```

Comment:
minor: style: I think that "enabled" suffix is redundant here, people will see the boolean type of the field and understand it just fine. E.g. `@Changefeed(virtualTimestamps = true, ...)`